mbta-card.js

```diff
@@ -58,7 +62,7 @@ class MBTACard extends HTMLElement {
             let limit = entity_dict["limit"];
             let attr = sensor["attributes"];
             let direction = attr['direction'];
-            let route = attr['route'].toLowerCase();
+            let route = (["red", "orange", "green", "blue"].includes(attr['route'].toLowerCase())) ? attr['route'].toLowerCase() : "commuter";
```

Comment:
Also any user-error or bugs will label the entity as "commuter" but that's not a big deal. I agree with you that this will be changed anyways to support buses and stuff